### PR TITLE
fix first load

### PR DESCRIPTION
**Pull request by ahmdshrif**
### Fixed Issues

$ https://github.com/Expensify/App/issues/7476

### Tests

- [x] Verify that no errors appear in the JS console

### QA Steps

1. Set phone language to any RTL ( Settings  -> Languages & input -> Languages ) chose Arabic for example 
2. remove the app and reinstall it. 
3. Open the app
4. Login & Browse
5. make sure the UI element is in its normal  position 
6. make sure the  normal text also does not align to right instead of left 
7. got to text input and use English keyboard (RTL language  not supported)
8. make sure text input aligns to left 
9. test on android and ios . and flow all previous steps. 

- [x] Verify that no errors appear in the JS console

### Tested On

- [x] Web
- [x] Mobile Web
- [x] Desktop
- [x] iOS
- [x] Android

#### Web

Screen Shot 2022-02-07 at 2 22 50 AM

#### Desktop

Screen Shot 2022-02-07 at 2 30 40 AM

#### iOS

Screen Shot 2022-02-07 at 2 02 02 AM
55b-d8534034caed.png)

https://user-images.githubusercontent.com/21364901/153756257-fa241dcb-c853-473f-bbe9-4d5c2bd2a2c5.mov

#### Android

Screen Shot 2022-02-07 at 1 45 17 AM

### CODE REVIEW

**Comment by marcaaron** [comment]
What's the context for this PR? Was there a regression somewhere?

**Comment by ahmdshrif** [comment]
I find the edge case here when you install the app for the first time app does not read the RTL config from js until you reload it . and will work after that very fine. 

the js config we add before. work fine with us because the app was installed.

**Comment by marcaaron** [comment]
Got it thanks!

File: ios/NewExpensify/AppDelegate.m [hunk 1]
```diff
@@ -37,6 +38,10 @@ - (BOOL)application:(UIApplication *)application didFinishLaunchingWithOptions:(
   // Configure firebase
   [FIRApp configure];
 
+  //force app to work lrt
```
Comment (parasharrajat) [inline ios/NewExpensify/AppDelegate.m:41]:
```suggestion
  // Force the app to LTR mode.
```

**Review by parasharrajat — CHANGES_REQUESTED** [review]
Thanks for submitting a PR @ahmdshrif Appreciate it.

**Review by parasharrajat — APPROVED** [review]
LGTM. 

cc: @marcaaron 

:ribbon: :eyes: :ribbon:  C+ reviewed

**Review by marcaaron — APPROVED** [review]

**Comment by mvtglobally** [comment]
Team is having a bit hard time validating this PR on Desktop/Web as it requires device language change and introducing navigation issues to flip language back as options shift.
We validated mWeb/IOS and Android. It would be great if someone can validate internally Desktop/Web

**Comment by ahmdshrif** [comment]
HI, @mvtglobally   I will suggest here to change the language for the app and browser **only**  from setting here. 


Screen Shot 2022-03-07 at 10 05 30 AM

**Comment by francoisl** [comment]
Hi @mvtglobally, were you able to test with the suggestion just above?

**Pull request merged**